spot_rl_experiments/spot_rl/envs/place_env.py

```diff
@@ -100,14 +100,22 @@ def __init__(self, config, spot: Spot):
         # Place steps
         self._time_step = 0
 
+        # Check whether its ee env
+        self.ee_env = False
+
     def decide_init_arm_joint(self, ee_orientation_at_grasping):
         """Decide the place location"""
 
         # User does not set the gripper orientation
         if ee_orientation_at_grasping is None:
-            self.initial_arm_joint_angles = np.deg2rad(
-                self.config.INITIAL_ARM_JOINT_ANGLES_SEMANTIC_PLACE
-            )
+            if self.is_ee_env:
+                self.initial_arm_joint_angles = np.deg2rad(
+                    self.config.INITIAL_ARM_JOINT_ANGLES_SEMANTIC_PLACE_EE
```

Comment:
let's make a `decide_init_arm_joint` function in `SpotSemanticPlaceEEEnv` to override this